src/app/modules/myinfo/myinfo.factory.ts

```diff
@@ -82,6 +91,30 @@ interface IMyInfoFactory {
   extractUinFin: (
     accessToken: string,
   ) => Result<string, MyInfoInvalidAccessTokenError | MissingFeatureError>
+
+  fetchMyInfoData: (
```

Comment:
I think we need to deconflict the naming for `fetchMyInfoData` and `fetchMyInfoPersonData`. we can also turn the latter into a private function if it doesn't need to be called externally. maybe can rename the latter to `callPersonEndpoint` or something

Reply:
i think given that the only callsite for `fetchMyInfoData` is in the middleware, i'll just delete the middleware and make it private.

not in favour of inlining it because it's a wrapper over the raw `Promise`, which should be made clear